web/wp-content/themes/dctx-v2/assets/bower_components/jquery/src/var/toString.js

```diff
@@ -0,0 +1,7 @@
+define( [
+	"./class2type"
+], function( class2type ) {
```

Comment:
:no_entry_sign: **Error**: Multi-line function call not indented correctly; expected 4 spaces but found 0 (*PEAR.Functions.FunctionCallSignature.Indent*).